src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -4905,6 +4916,9 @@ void ExportMusicXml::hairpin(Hairpin const* const hp, staff_idx_t staff, const F
             } else {
                 tag += "\"diminuendo\"";
             }
+            if (hp->lineColor() != engravingConfiguration()->defaultColor()) {
+                tag += QString(" color=\"%1\"").arg(QString::fromStdString(hp->lineColor().toString()).toUpper());
```

Comment:
Exporting hairpin color, but not importing it?

Reply:
I believe this could be done in color2xml(), check my backport to 3.x

Reply:
just noticed during testing that the existing implementation for glissando didn't work, so I fixed it. Happy to add import for hairpins.